webpack.common.js

```diff
@@ -1,11 +1,17 @@
 // @flow
 const path = require('path');
+// $FlowIgnore
 const TerserPlugin = require('terser-webpack-plugin');
+// $FlowIgnore
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+// $FlowIgnore
+const PnpWebpackPlugin = require('pnp-webpack-plugin');
 
 const {version} = require("./package.json");
 
+// $FlowIgnore
 const browserslist = require('browserslist')();
+// $FlowIgnore
 const caniuse = require('caniuse-lite');
```

Comment:
Why do we need `$FlowIngore`s for these imports now?

Reply:
@kevinbarabash See https://github.com/KaTeX/KaTeX/pull/2316#discussion_r456137525 above.

Reply:
Thanks.